tests/test_install.ts

```diff
@@ -69,6 +69,10 @@ const agamaSlowMo = parseInt(process.env.AGAMA_SLOWMO || '0');
 const agamaHeadless = booleanEnv("AGAMA_HEADLESS", true);
 const agamaInstall = booleanEnv("AGAMA_INSTALL", true);
 
+const agamaUser = "bernhard";
```

Comment:
could you please change `agamaPassword` to `nots3cr3t` ? I will help you for integrating with openQA.

Reply:
Done 895678b